macro/utils/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "iroha_macro_utils"
+version = "2.0.0-pre-rc.13"
+authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
+edition = "2021"
+description = "This crate contains various functions and structs for creating macros in Iroha project."
+readme = "README.md"
+homepage = "https://github.com/hyperledger/iroha/tree/iroha2-dev"
+repository = "https://github.com/hyperledger/iroha/tree/iroha2-dev"
+license = "Apache-2.0"
+keywords = ["crypto", "blockchain", "ledger", "iroha", "macro"]
+categories = ["cryptography::cryptocurrencies", "encoding", "development-tools::profiling", "development-tools::debugging"]
```

Comment:
This looks like a good practice to me, but I believe no other crates have this? Maybe creating a small issue for this cosmetic change across all Iroha would be warranted.

Reply:
You mean `categories`? If so we already have then in number of crates.